pkg/client/connector/k8s_config.go

```diff
@@ -21,6 +54,8 @@ type k8sConfig struct {
 	config      *rest.Config
 }
 
+const configExtension = "telepresence.getambassador.io"
```

Comment:
This should probably be `telepresence.io`? Since it's CNCF and not Ambassador Labs?

Reply:
We use `telpresence.getambassador.io` in other places. The annotation in the workloads for instance, is `telepresence.getambassador.io/actions`. I like the consistency, but perhaps we should consider changing the annotation too?

Reply:
Hmm, so we are.  I guess as long as that's what we're already doing, then it's good enough.